src/apps/relay/http_server.c

```diff
@@ -350,9 +350,9 @@ void str_buffer_append_sz(struct str_buffer *sb, size_t sz) {
   str_buffer_append(sb, ssz);
 }
 
-void str_buffer_append_sid(struct str_buffer *sb, turnsession_id sid) {
+void str_buffer_append_sid(struct str_buffer *sb, turnsession_id sid, uint32_t rsid) {
   char ssz[129];
```

Comment:
is this enough space to hold the new data in all cases?

Reply:
Yes. Actually 20 + 1 + 10 + 1 = 32 would be sufficient. Not sure, why the original author has chosen such a big buffer (doesn't make any sense to me).